docs/advanced_topics/customisation/admin_templates.rst

```diff
@@ -163,7 +163,7 @@ To add extra buttons to the login form, override the ``submit_buttons`` block. Y
 
     {% block submit_buttons %}
         {{ block.super }}
-        <a href="{% url 'signup' %}"><button type="button" class="button" tabindex="4">{% trans 'Sign up' %}</button></a>
+        <a href="{% url 'signup' %}"><button type="button" class="button">{% trans 'Sign up' %}</button></a>
```

Comment:
👍 good idea to update the docs to not use this pattern. It’s also pretty strange to have a `button` inside an `a` tag, but I’m not entirely sure what this is trying to demonstrate so won’t change that now.